tests/test_operations.py

```diff
@@ -137,3 +145,28 @@ def test_CRY_state10_controlqubit_0() -> None:
 def test_CRY_state01_controlqubit_0() -> None:
     result: torch.Tensor = operation.CRY(pi, state_01, (1, 0), 2)
     assert torch.allclose(state_11, result)
+
+
+def test_CSWAP_state000_controlqubit_0() -> None:
+    result: torch.Tensor = operation.CSWAP(state_000, (0, 1, 2), 3)
+    assert torch.allclose(state_000, result)
```

Comment:
the functional API of pyqtorch will be deprecated soon. can we add those tests to `tests/test_modules.py` rather than `test_operations.py`?

Reply:
oh yes, thanks @nmheim

Reply:
Maybe @nmheim (in a a separate PR) you can mark the functional API as deprecated? see e.g. Pulser for how to do that, or pasqal-cloud